### DISCUSSION TITLE
No systray on Debian 12

### DISCUSSION THREAD

**Discussion by legayf** [joshuar/go-hass-agent#275] [Q&A] [answered]
Hello,
When I run go-hass-agent I have this message : NFO systray error: failed to register: The name org.kde.StatusNotifierWatcher was not provided by any .service files. And no icon appear anywhere. 
And I can't edit configuration to enable MQTT (Go-hass-agent configure option seem to no longer exist).
Tnak-you for your help

**Comment by joshuar** [comment, marked as answer]
Hey I believe you need to install a Gnome Shell extension to get the system tray icon, for example [this one](https://extensions.gnome.org/extension/615/appindicator-support/). In my Debian 12 test VM, installing that extension allows the tray icon to be displayed.

**Reply by Typhus14** [reply]
Perfect ! It work fine. Thank you for the help and for all the work.
